scripts/acceptance.sh

```diff
@@ -2,16 +2,18 @@
 
 set -ex
 
-# Build the client assets for testing
-export BUILD_CLIENT=true
+if [ ! -f server.dist.js ]; then
+  # Build the client assets for testing
+  export BUILD_CLIENT=true
 
-# CSS is only compiled during a development build?
-export NODE_ENV=production
+  # CSS is only compiled during a development build?
+  export NODE_ENV=production
 
-yarn webpack
+  yarn webpack
+fi
 
 mocha \
   --retries 5 \
   --require test.config.js \
-  -t 60000 \
+  -t 360000 \
```

Comment:
6 minutes seems really long! Is anything taking that long?

Reply:
On 2nd thought, this matches my experience locally, where initial requests take 1-2 minutes to get a response. I assumed that was a local problem, or different with "production" assets.

Reply:
This timeout seems to be in place to allow the initial startup, its not really a "timeout". I'm sure that I can find a lower setting with a few extra run, I'll see what I can do.